submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
+  elem = document.createElement(elem);
```

Comment:
Okay, this code works, but you did a little mistake.

 Here you mutated parameter of your function.  In general you can do such things, but this is not that case.

Small explanation:
Firstly you pass `String`, but then you assigned `HTML element` (result of this operation  `document.createElement(elem);`)

Reply:
Good note. While it is possible to change the type of a variable in Javascript, it is best never to do so. I know, but I haven't looked. Fixed. Thank You.